modules/Hoot/ui/modals/importDataset.js

```diff
@@ -71,6 +69,17 @@ export default class ImportDataset {
         ];
     }
 
+    static importTypeMap() {
+        return {
+            'DIR': null,
+            'GEONAMES': '.geonames, .txt',
+            'OSM': '.osm, .osm.zip, .pbf',
+            'FILE': '.shp, .shx, .dbf, .prj, .zip',
+            'GEOJSON': '.geojson, .geo',
```

Comment:
is .geo really a valid extension for geojson?

Reply:
it was already there before so i assume someone put it for a good reason

google says "A GEO file is a 3D CAD file saved in the GEO format"

Reply:
I'd rather it be .json, can you make that switch?

Reply:
.geojson and .json

Reply:
fixed